setup_f5_k8s.sh

```diff
@@ -16,6 +16,7 @@ FORCE=0
 CUSTOM_MY_VALUES=()
 MY_VALUES=()
 DRY_RUN=""
+SOLR_REPLICAS=1
```

Comment:
`SOLR_REPLICAS` is a variable used below without default value, default to 1 here, let me know if it's intended to not have default value